src/main.c

```diff
@@ -35,7 +35,9 @@
 #include <pthread.h>
 #include <pwd.h>
 #include <signal.h>
+#ifndef __ANDROID__
 #include <spawn.h>
+#endif
```

Comment:
The same goes here as well.

Reply:
With your suggestions it builds for both Android and Linux desktop.  Shall I update the PR?